protocol/contracts/amo/Ramos.sol

```diff
@@ -370,12 +370,12 @@ contract Ramos is IRamos, TempleElevatedAccess, Pausable {
         uint256 feeAmt = quoteTokenAmountIn * rebalanceFees.rebalanceJoinFeeBps / BPS_PRECISION;
         if (feeAmt != 0) {
             emit FeeCollected(address(quoteToken), feeCollector, feeAmt);
-            quoteToken.safeTransfer(feeCollector, feeAmt);
+            tokenVault.borrowQuoteToken(feeAmt, feeCollector);
```

Comment:
This isn't right - the fees portion still needs to be sent to the fee collector.